backend/src/services/package/PackageService.ts

```diff
@@ -122,6 +130,22 @@ export class PackageService {
         }
     }
 
+    async getCost(packageId: string, dependencies: boolean)
+    {
+        let returnDict: { [key: string]: any } = {};
+        const packageCostService= new PackageCostService()
+        const standaloneCost = await packageCostService.getStandaloneCost(packageId);
+        if(!dependencies)
+        {
+            returnDict[packageId as string] = {totalCost : standaloneCost};
+            return returnDict;
+        }
+
+        const totalCost = await packageCostService.getCost(packageId);
+        returnDict[packageId as string] = {standaloneCost : standaloneCost, totalCost : totalCost};
```

Comment:
According to piazza the final json should include the standaloneCost and totalCost of all dependencies (recursively). So the format would be for a package A with dep B. And B has dep C:

{id_A: { standaloneCost: <costA>, totalCost: <costA+costB+costC>}, id_B: { standaloneCost: <costB>, totalCost: <costB+costC>}, id_C: {standaloneCost: <costC>, totalCost: <costC>}}